medcat/utils/relation_extraction/rel_dataset.py

```diff
@@ -18,7 +17,7 @@ class RelData(Dataset):
 
     log = logging.getLogger(__name__)
 
-    def __init__(self, tokenizer: TokenizerWrapperBERT, config: ConfigRelCAT, cdb: CDB = CDB()):
+    def __init__(self, tokenizer: Union[TokenizerWrapperBERT, TokenizerWrapperLlama, TokenizerWrapperModernBERT], config: ConfigRelCAT, cdb: CDB = CDB()):
```

Comment:
Again, protocol would help - see comment in line 96 in `rel_cat.RelCAT.__init__`